File: 3103/app/src/main/java/com/example/a3103/MainActivity.java
package com.example.a3103;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button buttonFirst = findViewById(R.id.button_first);
        Button buttonSecond = findViewById(R.id.button_second);
        Button buttonThird = findViewById(R.id.button_third);
        Button buttonFourth = findViewById(R.id.button_fourth);

        buttonFirst.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, FirstActivity.class)));
        buttonSecond.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, SecondActivity.class)));
        buttonThird.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, ThirdActivity.class)));
        buttonFourth.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, FourthActivity.class)));
    }
}

File: 2504/app/src/main/java/com/example/a2504/MainActivity.kt
package com.example.a2504

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttonFirstActivity = findViewById<Button>(R.id.buttonFirstActivity)
        val buttonSecondActivity = findViewById<Button>(R.id.buttonSecondActivity)

        buttonFirstActivity.setOnClickListener {
            val intent = Intent(this, FirstActivity::class.java)
            startActivity(intent)
        }

        buttonSecondActivity.setOnClickListener {
            val intent = Intent(this, SecondActivity::class.java)
            startActivity(intent)
        }
    }
}